src/Compilers/CSharp/Test/Syntax/Parsing/ParsingTests.cs

```diff
@@ -187,6 +187,7 @@ protected SyntaxNodeOrToken N(SyntaxKind kind, string? value = null)
             {
                 Assert.True(_treeEnumerator!.MoveNext());
                 Assert.Equal(kind, _treeEnumerator.Current.Kind());
+                Assert.False(_treeEnumerator.Current.IsMissing);
```

Comment:
This is the only real test change, the test are just fixing up tests that called `ParsingTests.N` when they should have been calling `ParsingTests.M`